app/src/main/java/org/xbmc/kore/ui/generic/SubtitlesOptionsDialog.java

```diff
@@ -0,0 +1,313 @@
+package org.xbmc.kore.ui.generic;
+
+
+import android.app.Activity;
+
+import android.os.Handler;
+import android.support.v4.app.FragmentActivity;
+import android.text.TextUtils;
+import android.widget.Toast;
+
+import org.xbmc.kore.R;
+import org.xbmc.kore.host.HostConnectionObserver;
+import org.xbmc.kore.host.HostInfo;
+import org.xbmc.kore.host.HostManager;
+import org.xbmc.kore.jsonrpc.ApiCallback;
+import org.xbmc.kore.jsonrpc.ApiMethod;
+import org.xbmc.kore.jsonrpc.HostConnection;
+import org.xbmc.kore.jsonrpc.method.Addons;
+import org.xbmc.kore.jsonrpc.method.GUI;
+import org.xbmc.kore.jsonrpc.method.Input;
+import org.xbmc.kore.jsonrpc.method.Player;
+import org.xbmc.kore.jsonrpc.type.ListType;
+import org.xbmc.kore.jsonrpc.type.PlayerType;
+import org.xbmc.kore.utils.LogUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * Dialog that presents a list of options related to subtitles.
+ */
+public class SubtitlesOptionsDialog
+        implements HostConnectionObserver.PlayerEventsObserver,
+                    GenericSelectDialog.GenericSelectDialogListener{
+
+
+    // An interface to be implemented by everyone interested in RemoteRequired events
+    public interface SubtitlesOptionsDialogListener {
+        void onRemoteRequired();
```

Comment:
Ah, I missed the implementation in NowPlayingPanel. Sorry about that.

https://github.com/xbmc/Kore/pull/427/files#diff-e3313a18ffdcf69828879c6a5097266eR50